docs/runbooks/monthly-release-to-editors-and-webmasters.md

```diff
@@ -46,8 +46,10 @@ release - 1. This usually mean last weeks release.
 15. Merge the pull request once the deployment completes.
 16. Run `task cluster:adjust:resource-request` from `dplsh`.
 17. Run `task cluster:promote-workloads-to-prod` from `dplsh`.
-18. Reset the webmaster moduletest environments.
-19. Run 'task cluster:mode:reset' from `dplsh`.
+18. Synchronize moduletest with main, so moduletest becomes an exact
+    copy of main by running `task sites:webmaster:reset-moduletest`.
+19. Reset the webmaster moduletest environments.
```

Comment:
What does this step 19 *Reset the webmaster moduletest environments* mean apart from what is already in step 18? Should it be deleted?

Reply:
done